cli/src/samples.rs

```diff
@@ -67,9 +67,8 @@ pub fn get_config_toml(
         .write(["sumeragi", "trusted_peers"], peers)
         .write(["network", "address"], DEFAULT_P2P_ADDR)
         .write(["network", "block_gossip_period_ms"], 500)
-        .write(["network", "block_gossip_max_size"], 1)
+        .write(["network", "block_gossip_size"], 1)
```

Comment:
I think it's pretty clear when combined with `gossip_period` that this is max value